src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -5,63 +5,115 @@ import moment from 'moment';
 import * as customPropTypes from '../prop-types';
 
 // Helper to derive the download link props.
-const deriveLinkProps = form => {
+const deriveLinkPropsFromFormURL = url => {
   const linkProps = {};
+  if (!url) return linkProps;
```

Comment:
Hm, worried about this because a search results without a URL isn't useful. How often do we see this?

Reply:
I don't know if we see it at all. I'm just fan of a guarded return.